src/main/java/core/basesyntax/Application.java

```diff
@@ -1,7 +1,13 @@
 package core.basesyntax;
 
 public class Application {
+
+    private static final Lottery lottery = new Lottery();
```

Comment:
Avoid using static fields for instances like `Lottery`. Instead, create an instance of `Lottery` in the `main` method, as static fields and methods are generally discouraged unless necessary (checklist item: Don't use static methods in your solution).